packages/components/tabs/src/tab-group.ts

```diff
@@ -215,10 +242,9 @@ export class TabGroup extends ScopedElementsMixin(LitElement) {
               </div>
             </div>
           </div>
-
           ${this.showMenu
             ? html`
-                <sl-menu-button aria-label=${msg('Show all')} fill="ghost">
+                <sl-menu-button aria-label=${msg('Show all')} fill="ghost" @keydown=${this.#onKeydown}>
```

Comment:
Minor nitpick, but please more `@keydown` to the start. I always try to put event handlers first, then boolean props, then props, then attrs.